### PR TITLE
Vetka1

### PR DESCRIPTION
**Pull request by Aliohin-Vladislav**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,5 @@
 # Project_FE
-## This is my first project!
\ No newline at end of file
+## This is my first project!
+*** hello world ***
```
Comment (IvanCeban) [inline README.md]: Can you please make it bold only?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,5 @@
...
+ergeg
```
Comment (IvanCeban) [inline README.md]: THis is redundant text. Can you remove it?

File: README.md [hunk 3]
```diff
@@ -1,2 +1,5 @@
...
+gergergerg
```
Comment (IvanCeban) [inline README.md]: THis is redundant also. Remove it please

**Review by IvanCeban — CHANGES_REQUESTED** [review]
Good job!

File: README.md (hunk 1 above)
Comment (Aliohin-Vladislav) [reply README.md]: zdelal

File: README.md (hunk 2 above)
Comment (Aliohin-Vladislav) [reply README.md]: zdelal

File: README.md (hunk 3 above)
Comment (Aliohin-Vladislav) [reply README.md]: zdelal

**Review by IvanCeban — APPROVED** [review]
Best job!

**Pull request merged**